setup.py

```diff
@@ -46,10 +46,11 @@ def get_name():
     keywords="fastestimator tensorflow",
     classifiers=[
         "License :: OSI Approved :: Apache Software License",
-        "Programming Language :: Python :: 3", ],
+        "Programming Language :: Python :: 3",
+    ],
 
     # Declare minimal set for installation
-    install_requires=['numpy'],
+    install_requires=['numpy', 'torch', 'tensorflow', 'albumentations'],
```

Comment:
torch should not be here, because `pip install torch` will not work for windows users.

for consistency, let's not put tensorflow here either.

Reply:
but pip install tensorflow will finally work now :(

Reply:
the torch website seems to think pip will work on windows